deliverable/build.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+set -x
+
+LAST_COMMIT=$(git rev-parse --short HEAD)
```

Comment:
Mejor con las variables en minúscula:
```suggestion
last_commit=$(git rev-parse --short HEAD)
```